jcl/src/java.base/share/classes/java/lang/J9VMInternals.java

```diff
@@ -185,6 +190,16 @@ private static void completeInitialization() {
 			}
 		}
 /*[ENDIF] CRAC_SUPPORT */
```

Comment:
While you're here, could you please indent lines 181 and 192 (`CRAC_SUPPORT`) to be consistent with the rest of this method.